Exercise_A/index.html

```diff
@@ -3,14 +3,13 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>Exercise_A</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="" />
+    <link rel="html" href="about.html" />
   </head>
   <body>
     <!-- CODE HERE -->
-
-    <script src="" async defer></script>
+    <a href="about.html"><img src="../img/teteum.png"></a>
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done!